src/ResourceManager/Search/Commands.Management.Search/help/Get-AzureRmSearchService.md

```diff
@@ -0,0 +1,141 @@
+---
+external help file: Microsoft.Azure.Commands.Management.Search.dll-Help.xml
+Module Name: AzureRM.Search
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmSearchService
+
+## SYNOPSIS
+Get an Azure Search service.
+
+## SYNTAX
+
+### ResourceGroupParameterSet
+```
+Get-AzureRmSearchService [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ResourceNameParameterSet
+```
+Get-AzureRmSearchService [-ResourceGroupName] <String> [-Name] <String>
```

Comment:
Be sure to regenerate this file after you make the parameter set changes to get-azurermsearchservice